src/mmrelay/radio/backends/meshtastic_backend.py

```
@@ -0,0 +1,320 @@
+                        if to_id and to_id != meshtastic.BROADCAST_ADDR
+                        else None
+                    ),
+                    message_id=message_id,
+                    reply_to_id=reply_to_id,
+                )
+
+                # Call user callback
+                try:
+                    callback(radio_message)
+                except Exception as e:
+                    backend_logger.exception(f"Error in radio message callback: {e}")
```

Ruff (0.14.14)

Redundant exception object included in `logging.exception` call

(TRY401)